frontend/src/types/PageTypes.ts

```diff
@@ -1,6 +1,9 @@
+import React from "react";
+
 interface Page {
   title: string;
   url: string;
+  icon?: () => React.ReactElement;
```

Comment:
Is there a particular reason why this is an optional field?

Reply:
+1! let's make this a mandatory field & maybe we can add a default icon in the `Sidebar` component!

Reply:
Oh I have it added like that because certain pages become subpages, which do not have icons

Reply:
oh jk you right! forgot about that